### PR TITLE
[Button] Size small height

### PR DESCRIPTION
**Pull request by jeremie-lucca**
36px > 32px

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1642

File: packages/scss/src/theming/components/_button.theme.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@ $button: (
 	padding: _theme('spacings.smaller') _theme('spacings.standard'),
 	small: (
 		font-size: _theme('sizes.small.font-size'),
-		line-height: _theme('sizes.small.line-height'),
+		line-height: _theme('sizes.smaller.line-height'),
```
Comment (MarcFairbrother) [inline packages/scss/src/theming/components/_button.theme.scss:10]: The small and smaller line-heights used to have the same value. Are we sure that we want them to have different values now?

Comment (jeremie-lucca) [reply packages/scss/src/theming/components/_button.theme.scss:10]: That's what we already do on .smaller : 
image
But to be honest I'm not even sure the theming should be linked to size variables!

Comment (MarcFairbrother) [reply packages/scss/src/theming/components/_button.theme.scss:10]: Ok, let's use this fix for now. It's a bit dodgy to use different size names so we should think about a more consistent solution for the future.

**Review by MarcFairbrother — APPROVED** [review]

**Pull request merged**